swift/ql/src/queries/Security/CWE-1204/StaticInitializationVector.ql

```diff
@@ -0,0 +1,72 @@
+/**
+ * @name Static initialization vector for encryption
+ * @description Using a static initialization vector (IV) for encryption is not secure. To maximize encryption and prevent dictionary attacks, IVs should rather be unique and unpredictable.
```

Comment:
It is true, because of the IV is hardcoded/constant, then it will suffice to just lookup potential keys in a dictionary, then concatenation those with the discovered IV rather than trying to discover the entire encryption key.

Reply:
Clarified that further in the `.qhelp` file.

Reply:
Thanks.